styles.js

```diff
@@ -9,6 +15,8 @@ export default createGlobalStyle`
 
   body {
     margin: 0;
-    font-family: system-ui;
+    font-family: ${goldman.style.fontFamily};
```

Comment:
I would recommend to not use this font as this is more a font that is suitable for headlines or decorative parts, but harder for the user if he needs to read the text. Try to use another font for the text.